### ISSUE TITLE
Spark app does not tile correctly

### ISSUE THREAD

**Issue by mwoffenden** [ianyh/Amethyst#952]
**Describe the bug**
The [Spark app](https://sparkmailapp.com/) is not tiling correctly in some situations.  See image below for details.

**Applications:**
[Spark app](https://sparkmailapp.com/)

**To Reproduce**
Tile Spark with at least one other app _and_ make sure Spark does not have focus.  Spark will tile but will be shifted UP.  See image for details.

**Screenshots**

![screenshot_3330](https://user-images.githubusercontent.com/6774346/74842933-263ba980-52f9-11ea-8e94-6221a76ed474.png)

**Versions:**
 - macOS: 10.14.6
 - Amethyst: 0.14.5b

**Additional context**
Spark tiles correctly if it's the only app on the screen, OR if it has the focus.

**Comment by ofilipowicz** [comment]
For me Amethyst just ignores Spark while tailing - Spark maintains it's original window's size and position, regardless of a layout chosen.

**Comment by mwoffenden** [comment]
UPDATE: More recently, Spark is tiling correctly for me.

**Comment by ofilipowicz** [comment]
I've just got back to using Amethyst and unfortunately the problem still exists. Spark doesn't tile at all. The same thing happens to ClickUp.

**Comment by mwoffenden** [comment]
@ofilipowicz Spark (2.11.15.866) is tiling well for me on Big Sur (11.6.5) with the latest version (0.15.6) of Amethyst.

**Comment by JuanG970** [comment]
To me, Spark (2.11.24.887) is just ignored on Monterey (12.6.1) with version 0.16.0 of Amethyst.

EDIT: Looks like it tiles well only when it is not assigned to "All Desktops" in the Dock:

<img width="163" alt="imagen" src="https://user-images.githubusercontent.com/36969954/205743814-f7ed8fe5-1739-46ff-b0cc-7aa5946a435b.png">

**Comment by ianyh** [comment]
I don't think this is a specific problem with Spark, and all dock assigned apps is in a different place.
